include/endpoint_lib.h

```diff
@@ -91,22 +91,6 @@ typedef void (*ava_deallocator)(void *ptr);
 
 //// Library functions/macros expected by metadata expressions
 
-#ifndef max
```

Comment:
Is there any reason for removing these lines?
(I can't really remember but they might be used in the customized QEMU.)